api/external/secrets/secrets_test.go

```diff
@@ -189,9 +189,9 @@ func TestValidateSecretFailures(t *testing.T) {
 	for _, test := range exampleValidationFailures {
 		err := test.in.Validate()
 		if err == nil {
-			t.Errorf("Sprintf(%q, validateSecret()) => nil, want %q", test.in, test.out)
+			t.Errorf("Sprintf(%q, validateSecret()) => nil, want %q", &test.in, test.out)
 		} else if err.Error() != test.out {
-			t.Errorf("Sprintf(%q, validateSecret()) => %q, want %q", test.in, err.Error(), test.out)
+			t.Errorf("Sprintf(%q, validateSecret()) => %q, want %q", &test.in, err.Error(), test.out)
```

Comment:
the `String()` method has a pointer receiver, dunno why it worked before and not now